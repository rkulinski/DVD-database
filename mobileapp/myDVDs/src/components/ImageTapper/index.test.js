import React from 'react';
import { ImageTapper } from './';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<ImageTapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
