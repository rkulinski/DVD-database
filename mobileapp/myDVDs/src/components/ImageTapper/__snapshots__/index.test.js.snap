// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "width": "48%",
      },
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Image
      reziseMode="stretch"
      source={undefined}
      style={
        Array [
          Object {
            "width": "100%",
          },
        ]
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "black",
            "paddingTop": 5,
            "textAlign": "center",
          },
        ]
      }
    >
      My first button
    </Text>
  </View>
</View>
`;
